tests/test_indexer.py

```diff
@@ -268,6 +268,17 @@ def test_subscription_notification_unsuccessful(self):
                          f"WARNING:.*:Failed notification for subscription {subscription_id}"
                          f" for bundle {bundle_id} with transaction id .+ Code: {error_response_code}")
 
+    def test_alias_exists(self):
+        sample_event = self.create_sample_bundle_created_event(self.bundle_key)
+        self.process_new_indexable_object(sample_event, logger)
+        es_client = ElasticsearchClient.get(logger)
+        index_names = [i['i'] for i in es_client.cat.aliases(name=self.dss_alias_name, h=['a', 'i'], format='json')]
+        self.assertNotEqual(len(index_names), 0)
```

Comment:
Thanks for adding the test. I think it should include verification that an alias with the expected name exists.
Any reason not to do that?

Reply:
The test does this. If the Alias did no exist then assertNotEqual would fail.

Reply:
Great, thanks!

Reply:
```diff
@@ -272,8 +272,7 @@
         sample_event = self.create_sample_bundle_created_event(self.bundle_key)
         self.process_new_indexable_object(sample_event, logger)
         es_client = ElasticsearchClient.get(logger)
-        index_names = [i['i'] for i in es_client.cat.aliases(name=self.dss_alias_name, h=['a', 'i'], format='json')]
-        self.assertNotEqual(len(index_names), 0)
+        self.assertTrue(es_client.indices.exists_alias(name=[self.dss_alias_name]))
 
     @unittest.skip("WIP")
     def test_index_when_multiple_indexes_using_alias(self):
```

I think this is simpler, more expressive and stricter, too.